utils/paginator.py

```diff
@@ -0,0 +1,191 @@
+import asyncio
+import discord
+from discord.ext.commands import Paginator as BasePaginator
+from discord.ext import commands
+import typing as t
+from discord.ext.commands import Bot, Context
```

Comment:
```suggestion
import asyncio
import typing as t

import discord
from discord.ext import commands
from discord.ext.commands import Bot, Context
from discord.ext.commands import Paginator as BasePaginator
```
These imports need to be properly sorted
<details>
<summary>you can use vscode's functionality for it as seen in this GIF</summary>
<p>

![Peek 2020-06-18 10-01](https://user-images.githubusercontent.com/20902250/84994237-b32d2d00-b14a-11ea-8f87-571aad829132.gif)
</p>
</details>